client/state/data-layer/wpcom/read/following/mine/delete/index.js

```diff
@@ -41,8 +61,8 @@ export function receiveUnfollow( store, action, next, response ) {
 export function unfollowError( { dispatch }, action, next ) {
 	dispatch(
 		errorNotice(
-			translate( 'Sorry, there was a problem unfollowing that site. Please try again.' )
-		)
+			translate( 'Sorry, there was a problem unfollowing that site. Please try again.' ),
```

Comment:
I think it would be good to specify which site had the error.
If you unfollow a couple in sequence it would stink to not know which one failed